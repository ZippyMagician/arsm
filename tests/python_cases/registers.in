This will be summed up