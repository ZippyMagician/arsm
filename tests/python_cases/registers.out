188,220,151,213,140,199,147,226,210,132,229,